webui/react/src/pages/ExperimentDetails/ExperimentSingleTrialTabs.tsx

```diff
@@ -77,12 +78,16 @@ const ExperimentSingleTrialTabs: React.FC<Props> = (
 
   const fetchFirstTrialId = useCallback(async () => {
     try {
+      // make sure the experiment is in terminal state before the request is made.
+      const isTerminalExp = terminalRunStates.has(experiment.state);
       const expTrials = await getExpTrials(
         { id: experiment.id, limit: 2 },
         { signal: canceler.signal },
       );
       if (expTrials.trials[0]) {
         setFirstTrialId(expTrials.trials[0].id);
+      } else if (isTerminalExp) {
```

Comment:
**question:** should this also check to see if `experiment.state` is `PAUSED`?

Reply:
discussed async, we don't want the polling to stop if experiment is paused in the case the experiment is resumed externally (outside of this page).